client/src/navigation/index.tsx

```diff
@@ -37,8 +37,6 @@ export const Navigation = (props: PropsWithChildren<NavigationProps>) => {
     <div
       css={css`
         background-color: ${pinboard[500]};
-        border-top-left-radius: 4px;
-        border-top-right-radius: 4px;
```

Comment:
did we want to make the corners square?

Reply:
no, I was being a bit lazy and didn't want to spend too much time making the radius play nicely with the corner the callout shape was sat on (because the radius on that corner looks horrible) - but that was naughty, thanks for keeping me honest. 
I've now got it working nicely and actually made it so all three corners which don't have the callout on them...
| | **Left** | **Right** |
| --- | --- | --- |
| **Top** | <img width="390" alt="image" src="https://user-images.githubusercontent.com/19289579/184020348-fdcbb989-45b3-4952-b818-17a312647326.png"> | <img width="387" alt="image" src="https://user-images.githubusercontent.com/19289579/184020135-3bbce95e-8500-4d70-b9cd-e89ed8800846.png"> |
| **Bottom** | <img width="395" alt="image" src="https://user-images.githubusercontent.com/19289579/184020494-2ca29f63-1a86-405b-9ae2-64bcbdba4839.png"> | <img width="402" alt="image" src="https://user-images.githubusercontent.com/19289579/184020182-d8648ed8-d87c-439a-a883-90007671ad1d.png"> |

Unfortunately `Navigation` also requires the radius, which meant passing down `isTopHalf` and `isLeftHalf` as props to `Navigation`.